libparsec/crates/platform_storage/src/web/workspace.rs

```diff
@@ -140,6 +143,16 @@ impl PlatformWorkspaceStorage {
         )
     }
 
+    pub async fn list_manifests(
+        &mut self,
+        page: u32,
+        limit: u32,
+    ) -> anyhow::Result<Vec<RawEncryptedManifest>> {
+        Vlob::list(&self.conn, page, limit)
+            .await
+            .map(|vlobs| vlobs.into_iter().map(|vlob| vlob.blob.to_vec()).collect())
```

Comment:
```suggestion
.map(|vlobs| vlobs.into_iter().map(|vlob| Vec::<u8>::from(vlob.blob)).collect())
```
This should allows us to do zero copy given the `Bytes` object is only referenced once here (see https://github.com/tokio-rs/bytes/issues/711#issuecomment-2150558312)

Reply:
And as a side note: is there a reason to use `Bytes` types in the web model ?

I'm under the impression we are always passing `Vec<u8>` types (given the caller is passing data that have been serialized for the only purpose of being stored in the database) and returning a `Vec<u8>` is also okay (again, the caller will only do de-serialization of those data then drop the buffer).

So should we create an issue about converting those `Bytes` into `Vec<u8>` ?

Reply:
done